test/x86/avx512f.c

```diff
@@ -1309,6 +1534,214 @@ test_simde_mm512_setone_si512(const MunitParameter params[], void* data) {
   return MUNIT_OK;
 }
 
+static MunitResult
+test_simde_mm512_srli_epi64(const MunitParameter params[], void* data) {
+  (void) params;
+  (void) data;
+
+  const struct {
+    simde__m512i a;
+    uint32_t     b;
+    simde__m512i r;
+  } test_vec[8] = {
+    { simde_x_mm512_set_epu64(UINT64_C(0x3d7f1878959e7823), UINT64_C(0xdb19c9e23de5574c),
+                              UINT64_C(0x41f761e519d11af5), UINT64_C(0x87862bc5e638d1f3),
+                              UINT64_C(0x48d3b224fa800458), UINT64_C(0xa540b9aff87ed332),
+                              UINT64_C(0x171283b07eae579a), UINT64_C(0xf871de8b316995f7)),
+      UINT32_C(0),
```

Comment:
Again, the type should be `unsigned int` not `uint32_t`, so these should just be `xU` instead of `UINT32_C(x)`.

Reply:
OK, I'll correct.
I didn't understand the purpose of these macros.
When I sent a raw value in another PR, someone told me to add a UINT64_C() so I thought they were mandatory.

Reply:
They're from the C standard, for portability.  See § 7.20.4.1 of C17, but they go back to C99.  They are mandatory for fixed-length types, but should not be used for int/short/long/char/long long/etc.

Think of them as adding, say, U or L suffixes.  When declaring an `unsigned long` with a value of 1729 in your code, you should use `1729UL` (it's more important for larger values, but you get the idea).  However, a `uint32_t` may be mapped to unsigned long, unsigned int, unsigned long long, etc., so for portability we need a macro which appends the correct suffix.  If `uint32_t` is compatible with `unsigned int`, `UINT32_C(1729)` is equivalent to `1729U`.  If it's compatible with `unsigned long`, then it's equivalent to `1729UL`, and so on.

Reply:
Thanks.
C99 ??? C17 ??? I'm still on K&R ;-) No only joking !

Reply:
MSVC still doesn't fully support C99.  They only added inttypes.h and stdint.h in 2013, and there are actually a lot of people still using old MSVC versions… upgrading a proprietary compiler which costs a significant amount of money isn't a very attractive idea for a lot of people.